ui/components/Navigator.js

```diff
@@ -747,10 +747,6 @@ class Navigator extends React.Component {
     category = category.toLowerCase();
     meshAdapters.forEach((adapter) => {
       let aName = adapter.name.toLowerCase();
-      // Manually changing adapter name so that it matches the internal name
-      if (aName === "osm") aName = "open service mesh";
-      if (aName === "cilium") aName = "cilium service mesh";
-      if (aName === "nginx") aName = "nginx service mesh";
```

Comment:
For open and nginx service mesh, adapter name as received in response is `open_service_mesh` and `nginx_service_mesh` so this check is redundant.
For the cilium adapter, `id` above was set as `cilium` which after this check is changed to `cilium service mesh` causing the undesired behaviour. Server response id `cilium_service_mesh`